### DISCUSSION TITLE
Shop pack grouping option (dashboard)

### DISCUSSION THREAD

**Discussion by CrimeFreeArk** [gameserverapp/Platform#958] [Suggestions]
We are using GSA for ASA and this is relating to the Shop Items in the dashboard.

Love the GSApp and its been super great.  One thing that is a bit difficult to manage though, is the shop items back-end.  We are at several hundred items now I think?  And it would be great if there was a way to manage the back-end to make it easier to find things rather than scrolling down to them or searching each time, as we regularly want to open several tabs of them and go back and forth when making adjustments.

We tried to use labels to sort them, but it still leaves the overall list seeming a bit overwhelming.

Is it possible to get some sort of folder system, where we could just sort them so we could say go and look at all of the dinos in one folder, all of the subscription packs in another folder, etc.  Obviously labels or something else would allow them to be sorted to the folders, but it is more of a UI/UX thing, for the dashboard.

**Comment by mvaessen** [comment]
hi, the sidebar should offer you options to filter items. 
![Screenshot 2024-11-13 at 12 51 11](https://github.com/user-attachments/assets/65013c9c-87f0-47ce-b47e-718e9299d278)

**Comment by CrimeFreeArk** [comment]
Yes but that's a temporary filter that resets after every action.   We are
taking about more of a permanent sorting system.

**Comment by mvaessen** [comment]
Thanks for your suggestion! This is now available on the dashboard
